digitaloceanmodule.php

```diff
@@ -3,7 +3,7 @@
 class Digitaloceanmodule extends Module {
 
     private static $version = "2.2";
-    private static $authors = array(array('name' => "ModulesBakery.com.", 'url' => "http://www.modulesbakery.com"));
+    private static $authors = array(array('name' => "ModulesBakery.com.", 'url' => "http://www.modulesbakery.com"),array('name' => "CubeData", 'url' => "https://cubedata.net"));
```

Comment:
Can you add a space between the current author array and your array? That, or break up the array to limit line length, e.g.

```
private static $authors = array(
    array(
        'name' => ...
        'url' => ...
    ),
    array(
        'name' => ...
        'url' => ...
    )
);
```